lib/postfix_version.rb

```diff
@@ -0,0 +1,7 @@
+if  FileTest.exists?("/etc/postfix/main.cf")
+	Facter.add("postfix_version") do
+		setcode do
+			%x{postconf mail_version | awk '{print $3}'}
+		end
```

Comment:
Either use the command as a parameter to `setcode`:

```suggestion
		setcode "postconf mail_version | awk '{print $3}'"
```

or parse using Ruby:

```suggestion
		setcode do
			Facter::Core::Execution.execute('postconf mail_version').split[4]
		end
```